app/upgrades/signetlaunch/upgrades_test.go

```diff
@@ -7,14 +7,18 @@ import (
 
 	"cosmossdk.io/core/appmodule"
 	"cosmossdk.io/core/header"
+	sdkmath "cosmossdk.io/math"
 	"cosmossdk.io/x/upgrade"
 	upgradetypes "cosmossdk.io/x/upgrade/types"
+	appparams "github.com/babylonlabs-io/babylon/app/params"
+	"github.com/babylonlabs-io/babylon/test/e2e/util"
 	tmproto "github.com/cometbft/cometbft/proto/tendermint/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
 	"github.com/stretchr/testify/suite"
 
 	"github.com/babylonlabs-io/babylon/app"
-	v1 "github.com/babylonlabs-io/babylon/app/upgrades/signetlaunch"
+	"github.com/babylonlabs-io/babylon/app/upgrades/signetlaunch"
```

Comment:
maybe it is time to re-name `signetlaunch` to standard `v1` naming as in other chains ? (i.e https://github.com/osmosis-labs/osmosis/tree/main/app/upgrades) 

I imagine phase-2 upgrade will be v1, phase-3 will be v2, ... etc. ?

wdyt ? (it can be done in separate pr though)

Reply:
sounds good, I will rename the package

Reply:
renamed to v1 at https://github.com/babylonlabs-io/babylon/pull/103/commits/b63978f3a8e45a997c779b3d8d58753e6fa19790